helm/v1beta1/types_helm.go

```diff
@@ -36,6 +36,9 @@ type HelmChartRepositoryList struct {
 // Helm chart repository exposed within the cluster
 type HelmChartRepositorySpec struct {
 
+	// If set to true, disable the repo usage in the cluster
```

Comment:
```suggestion
	// If set to true, disable the repo usage in the cluster
	// +kubebuilder:validation:Optional
	// +optional
```

Reply:
`kubebuilder:validation:Optional` can be specified only on package:
 https://book.kubebuilder.io/reference/markers/crd-validation.html

Reply:
that's the default already https://github.com/openshift/api/blob/master/helm/v1beta1/doc.go#L5